pkg/virt-api/rest/streamer_test.go

```diff
@@ -148,38 +149,52 @@ var _ = Describe("Streamer", func() {
 		Expect(dialCalled).To(BeFalse())
 	})
 	It("upgrades the client connection", func() {
+		var wg sync.WaitGroup
+		wg.Add(1)
 		srv, ws, wsResp, err := testWebsocketDial(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+			defer wg.Done()
```

Comment:
missing `defer GinkgoRecover()`?

Reply:
No, since `GinkgoRecover` is only necessary when calling Gomega functions, since they use panics to stop code execution on fail, so you need to call it manually in all goroutines you start yourself for the test.
this function does not call the functions, so it does not need to call `defer GinkgoRecover()`